### DISCUSSION TITLE
The Discord Server

### DISCUSSION THREAD

**Discussion by kayhantolga** [betalgo/openai#598] [Announcements]
I've set up a Discord server. Based on my recent survey, I'm not expecting a large number of developers. However, if you're interested, you can join using this link: https://discord.gg/rHEu5THg. If the link is not working, please notify me here, and I'll generate a new one.

**Comment by odeta939** [comment]
Hey! I am new to OpenAI, I would love to join the discord server! Could I get a new invite link as the old one is invalid

**Reply by kayhantolga** [reply]
Hi, welcome! https://discord.gg/rE9uVp52
